python/tests/integration/test_websocket_server_integration.py

```
@@ -2,33 +2,14 @@
 
 import pytest
 
+from .apps import app_source
 from .helpers import SimpleWebSocketClient
 
 pytestmark = pytest.mark.server_integration
 
 
 def _make_websocket_project(make_server_project):
```

Ruff (0.15.17)

[warning] Missing return type annotation for private function `_make_websocket_project`

(ANN202)